src/components/App.jsx

```diff
@@ -31,8 +31,7 @@ class App extends Component {
           <div className="view">
             {React.createElement(view, viewProps)}
           </div>
-          <SideBar />
-
+          <SideBar songs={this.props.library} dispatch={this.props.dispatch} />
```

Comment:
penso che l'utente si aspetti che aggiungendo della musica nel lettore questa vada automaticamente anche nella coda di riproduzione (molti lettori funzionano cosi)
In ogni caso per il momento può andare come stub